### PR TITLE
Modules installing to prepare workflow submission

### PR DESCRIPTION
**Pull request by carpanz**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/hgtseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/hgtseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by lescai — CHANGES_REQUESTED** [review]
I world recommend removing the local ranalysis module from this PR, so this is only about installing nf-core tested modules and we can focus one of the following PRs to proper code review of each local module(s)

**Comment by carpanz** [comment]
Done! @lescai

File: modules.json [hunk 1]
```diff
@@ -3,15 +3,42 @@
...
+            "samtools/view": {
+                "git_sha": "6b64f9cb6c3dd3577931cc3cd032d6fb730000ce"
+            },
+            "trimgalore": {
+                "git_sha": "85ec13ff1fc2196c5a507ea497de468101baabed"
             }
         }
     }
-}
+}
```
Comment (lescai) [inline modules.json:44]:
```suggestion
}

```

**Review by lescai — CHANGES_REQUESTED** [review]
json files (modules.json failing Prettier) seem to need a new line character at the end of the file to be valid

**Review by lescai — APPROVED** [review]
LGTM now. I will skip last test until we get subworkflows we can actually test

**Pull request merged**